### PR TITLE
Display weather based on visitor geolocation

### PR DESCRIPTION
**Pull request by EevanR**
[PT-LINK](https://www.pivotaltracker.com/story/show/171034912)

User Story

As a user
In order to stay up to date with the weather
I want to see realtime weather information

- display weather information from API

### CODE REVIEW

File: src/components/Weather.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  useEffect(() => {
+    getLocation()
+  }, []);
+
+  useEffect(() => {
+  loadWeatherData()
+  }, [coords]);
+  
+  return (
+      <div className="weatherMain">
```
Comment (mantanquzzmo) [inline src/components/Weather.jsx]: weather-main

File: cypress/integration/weather.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/// <reference types="Cypress" />
+
+describe("Weather div displays", () => {
+  it("weather info", () => {
+    cy.visit("/");
+    cy.get(".weatherMain").should("contain", 1.9);
```
Comment (mantanquzzmo) [inline cypress/integration/weather.js]: Use only one get.

File: src/components/Weather.jsx (hunk 1 above)
Comment (EevanR) [reply src/components/Weather.jsx]: thought kebab case was only for id's..?

File: src/locales/default_sv.json [hunk 3]
```diff
@@ -55,6 +55,10 @@
       "undo": "Ångra",
       "undidMess": "Ångrad publicering"
     },
+    "weather": {
```
Comment (mantanquzzmo) [inline src/locales/default_sv.json]: Maybe use Temp or Temperatur instead of nu. Nu feels a bit weird :)

Comment (EevanR) [reply src/locales/default_sv.json]: took it away

File: src/components/Weather.jsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      setCoords([loc.coords.latitude, loc.coords.longitude])
+    })
+  }
+
+  useEffect(() => {
+    getLocation()
+  }, []);
+
+  useEffect(() => {
+  loadWeatherData()
```
Comment (sundqvistjohan) [inline src/components/Weather.jsx]: indentation

**Pull request merged**